src/main/java/domain/LottoMatch.java

```diff
@@ -0,0 +1,43 @@
+package domain;
+
+import java.util.Arrays;
+
+public enum LottoMatch {
+  BOOM(0, false, 0),
+  FIFTH_PLACE(3, false, 5000),
+  FOURTH_PLACE(4, false, 50000),
+  THIRD_PLACE(5, false, 1500000),
+  SECOND_PLACE(5, true, 30000000),
+  FIRST_PLACE(6, false, 2000000000);
```

Comment:
숫자가 길어지면 이 값이 얼마인지 한눈에 파악하기가 어려워지는 것 같아요. 
자바는 언어단에서 3자리 숫자마자 `_`를 붙일 수 있도록 지원해주는데요. 

아래와 같이 `_`로 숫자의 가독성도 높여주면 좋을 것 같아요. 😃 
```suggestion
  FIFTH_PLACE(3, false, 5_000),
  FOURTH_PLACE(4, false, 50_000),
  THIRD_PLACE(5, false, 1_500_000),
  SECOND_PLACE(5, true, 30_000_000),
  FIRST_PLACE(6, false, 2_000_000_000);
```

Reply:
,와 같은 기능을 코드 상에서도 표현할 수 있다니! 감사합니다.